force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -5858,51 +5858,43 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
     </labels>
     <labels>
         <fullName>commonAmount</fullName>
-        <categories>Gift Entry</categories>
+        <categories>Recurring Donations</categories>
```

Comment:
`commonAmount` is an already existing custom label and is also being used in Gift Entry. You can leave the categories unchanged.